src/Css/Stylesheet.php

```diff
@@ -965,6 +966,11 @@ function apply_styles(FrameTree $tree)
 
                 /** @var \DOMElement $node */
                 foreach ($nodes as $node) {
+                    // Expect $node to be an instance of DOMElement class
+                    if (!$node instanceof DOMElement) {
```

Comment:
Shouldn't we wrap the statement in parenthesis (`if (!($node instanceof DOMElement)) {...}`?

Reply:
Not really, as far as I know. It works fine for me and @akaunting users.

Reply:
You're right, in the [operator precedence documentation](http://php.net/manual/en/language.operators.precedence.php) `instanceof` comes before the negation operator. I guess not having the forced precedence makes me a bit uncomfortable. However, as you stated, not required.